src/components/Parentchild/index.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import { Button, Divider,Stack } from '@mui/material';
+import DividerTag from '../atoms/Divider';
+import TextButton1 from '../molecules/Textbutton';
+import Slider1 from '../atoms/Slider';
+import Typography1 from '../atoms/Typography';
+import Textdivider1 from '../molecules/Textdivider';
+import AvatarText1 from '../molecules/Avatartext';
+import theme from '../../theme';
+import Statemgmnt from '../organism/Statemanagement';
+
+const Parentchild =()=> {
+
+    const [isClicked, setIsToggled] = React.useState(false);
+
+//     const [value, setvalue] =React.useState(0.0234510)
+//     const handleChange =(event: any, newValue: React.SetStateAction<any>) => {
+//       setvalue(newValue)
+//     }
+// console.log({value});
+    return (
+        <Box>
+            <Stack component="span" 
+                sx={{
+                    border:'2px solid whitesmoke',
+                    mx:50,
+                    height:250,
+                    borderRadius:1,
+                    mt:5,
+                }}>
+                 <Typography variant="subtitle1" gutterBottom sx={{pl:2}}>Amount details</Typography> 
+            <TextButton1 buttonVariant={'outlined'} buttonName={'Buy Max'} typoVariant={'h6'} typoName={'$34,000.00'}
+            typoSx={{ paddingLeft: 3, paddingTop: 2 }} 
+            buttonSx={{color:"theme.palette.primary.main",fontWeight:"bold",maxWidth:100,position:"absolute",marginLeft:63,marginTop:1.4,textTransform:"none"}}/>
+
+            {/* <Slider1 value={value} handleChange={handleChange} /> */}
+            <Slider1 />
+            <TextButton1 buttonVariant={'text'} buttonName={'BTC'} typoVariant={'h6'} typoName={"0.0234510"} 
+            typoSx={{ paddingLeft: 3, paddingTop:1.5 }} 
+            buttonSx={{marginLeft:68,position:"absolute",marginTop:1,color:"grey",fontWeight:"bold"}} /> 
+
+            </Stack>     
+
+            <Box
+                sx={{
+                    boxShadow:1,
+                    ml:50,
+                    mt:2,
+                    width: '670px',
+                    height: '500px',
+                   borderColor:"whitesmoke",
+                   borderLeft:1,
+                   borderLeftColor:'whitesmoke',
+                   borderBottom:1,
+                   borderBottomColor:'whitesmoke',
```

Comment:
you are already giving borderColor, why again border-left and right?

Reply:
ok I removed it.